lib/moments.rb

```diff
@@ -8,4 +8,8 @@ module Moments
   def self.difference(from, to, mode = :normal)
     Moments::Difference.new from, to, mode
   end
+
+  def self.ago(from, mode = :normal)
+    Moments::Difference.new from, Time.now, mode
+  end
```

Comment:
It can conflict (not account) with Rails timezones.

Reply:
IMHO if you are using Rails there are plenty of other options to perform Time calculations. The beauty of this gem is its simplicity and independence of any other gem.

The ago method is just a shortcut and I do not see what new issues this causes. If you do not want to calculate against Time.now, feel free to use Difference.new at any time. But if you add a test case that proves your point, I will gladly look into it.

Reply:
Yeah. For now, it's just a note from me to think and be aware.

Different, non-Rails, projects can use something for timezones too. Especially with databases…